run.sh

```diff
@@ -80,18 +78,18 @@ pid=$!
 count=0
 wid=""
 
-while [ -z "$wid" ]; do
+while [[ -z "$HEADLESS" && -z "$wid" ]]; do
     wid=$(wmctrl -l | grep " Google Chrome" | cut -f 1 -d ' ')
     if [ -n "$wid" ]; then
         echo "Chrome Found"
         break
     fi
     sleep 0.5
-    count=$[$count + 1]
+    count=$((count+1))
```

Comment:
this could be `((count+=1))` if you wanted to simplify this statement

Reply:
Thanks, updated